src/AppBanner/canRender.ts

```diff
@@ -0,0 +1,16 @@
+import type { BrazeMessageProps } from './index';
+import { isImageUrlAllowed } from '../utils/images';
+
+export const COMPONENT_NAME = 'AppBanner';
+
+export const canRender = (brazeMessageProps: BrazeMessageProps): boolean => {
```

Comment:
I'm not sure where the right place is, but it feels like a comment explaining why this is in a separate file would be useful.

Reply:
Added a comment to all canRender files